<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UAB</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="create_project_button">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>300</y>
      <width>561</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create Project</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="project_name">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>561</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Project Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="project_path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>561</width>
      <height>32</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Project Path</string>
    </property>
   </widget>
   <widget class="QPushButton" name="browse_button">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>130</y>
      <width>75</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="description">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>561</width>
      <height>100</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Project Description</string>
    </property>
   </widget>
   <widget class="QLabel" name="status">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>561</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: Ready</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>